### PR TITLE
Dorika j patch 1

### PR DESCRIPTION
**Pull request by DorikaJ**
Initial version of the HTML presentation and notebook (images were not uploaded, but added to HTML slides)

### CODE REVIEW

**Comment by friedgab** [comment]
Let's not mention MQA in the filename.

**Comment by friedgab** [comment]
Apologies, I cannot make comments on the diffs as it's too large to display, so I'll just leave my comments here...

Typo on the slide starting with "According to 1. and 2.":
is the **pricde** per unit of asset

**Comment by friedgab** [comment]
On the slide starting with "In some cases we were":

- There's an image, that is not commited, hence, the link is broken: images/**linearprice.png**. I think in general you should commit all images as well. You may want to create a folder for your presentation and put everything in there.
- Typo: **For this we will a model for the asset price.**

**Comment by friedgab** [comment]
On the slide starting with "What is stochastic process?":
- random variables X(t) indexed by a parameter t. Let's use the **notation** X_t if it's later referenced as X_0, X_1, etc and not as X(0), X(1), etc.

**Comment by friedgab** [comment]
Broken image link on first slide: images/invest.png. Please commit the image file as well.

**Comment by DorikaJ** [comment]
Are the images missing from the HTML files as well?

**Comment by friedgab** [comment]
> Are the images missing from the HTML file as well?

Yes. Though not all of them. E.g. I can see  on slide: Further properties.
Better to save everything and commit in the repo.

**Comment by friedgab** [comment]
On Slide: Back to coin tossing..
- Why is it called back to? Do you plan to mention it earlier?
- "In the binary tree models we have seen the idea of" - I don't think everybody has experience to binary tree models. I would rephrase this slide a bit.

**Comment by friedgab** [comment]
On the slide which defines `randomwalk1D`:
- On the slide before I would use j as index, not t as you mentioned.
- The implementation is not consistent with the notations before: y is already a cumulative sum. Also you can use the timepoints range that you created before for looping . I would create y as a list, and add +1 or -1 in them, then at the end use `np.cumsum()` and name it S.
- `rw_paths2=np.concatenate([origin, steps]).cumsum(axis=0)` Neither origin, nor steps are defined, hence, throwing error.

**Comment by DorikaJ** [comment]
Apologies, I thought the images will be added once the HTML is generated. I am currently working on it, the indexing needs to be fixed and some typos. Uploaded the images and the latest version of the files.

**Comment by friedgab** [comment]
`[variance(path) for path in M[1:10]]` - Misleading. Here you don't loop through the paths, but the time slices.

**Comment by friedgab** [comment]
On the slide defining Brownian motion: Let's use index everywhere. B_t and B_0 (not B(0)).

**Comment by friedgab** [comment]
Typo - "We have to scale down or **shink** the size of jumps."

**Comment by friedgab** [comment]
Brownian motion has a continuous path so I would not use the world `jump` on any slides. A few possibly better words coming into my mind: move, movement, change, variation.

**Comment by friedgab** [comment]
Typo - "**Addd** a term to the Brownian motion"

**Comment by friedgab** [comment]
I have a feeling that the last part will be super heavy for most of the participants. I also have concerns if it will fit in 90 mins, but let's see the feedbacks after the presentation. :)

**Comment by DorikaJ** [comment]
Thanks for the feedback. Tomorrow I will make the adjustments, do a trial on time and maybe we can discuss what can be removed or might be too much on this level.

**Review by friedgab — APPROVED** [review]
